server/courselist.rb

```diff
@@ -627,6 +627,22 @@ def run_url_parse_json(new_url)
       logger.debug "#{__method__}: #{__LINE__}: #{new_url}: request_body_ruby: +++#{request_body_ruby.inspect}+++"
       request_body_ruby
     end
+
+    # Get the canvas courses from the list provided by mpathways.
+
+    def getCanvasCourseList(course_data, userid)
+      # could the regexp be a constant?
```

Comment:
I'll improve the name and, due to deadlines, make a jira to make the regexp a constant in a later release.